src/api/decorators.py

```diff
@@ -41,5 +22,29 @@ def handle_feedback(request, region_slug, language_code):
             return JsonResponse(
                 {"error": f'No language found with code "{language_code}"'}, status=404
             )
+        if request.content_type == "application/json":
+            data = json.loads(request.body.decode())
+        else:
+            data = {}
+            for key in request.POST:
+                data[key] = request.POST.get(key)
```

Comment:
The for-loop is redundant.
Just use `data = request.POST.dict()`

Reply:
Does not work for me that way.

Reply:
Okay, tried again. It does work. There was probably some other reason for the issues before.